packages/app/src/components/uiBlocks/Dropdown.vue

```diff
@@ -103,76 +96,53 @@ export default defineComponent({
       required: false,
       default: '',
     },
+    items: {
+      type: Array as PropType<ListItem[]>,
+      default: (): ListItem[] => [],
```

Comment:
does this syntax creates an new instace?

Reply:
yeah, a new empty array of type ListItem.